res/controllers/hid-controller-api.d.ts

```diff
@@ -7,22 +7,22 @@ declare namespace controller {
      * @param dataList Data to send as list of bytes
      * @param length This optional argument is no longer evaluated, and only here for backwards compatibility [default = 0]
     */
-    function send(dataList:number[], length?:number):void;
+    function send(dataList: number[], length?: number): void;
 
     /** Sends HID OutputReport to HID device
      * @param dataList Data to send as list of bytes
      * @param length Unused but mandatory argument
      * @param reportID 1...255 for HID devices that uses ReportIDs - or 0 for devices, which don't use ReportIDs
      * @param resendUnchangedReport If set, the report will also be send, if the data are unchanged since last sending [default = false]
     */
-    function send(dataList:number[], length:number, reportID:number, resendUnchangedReport?:boolean):void;
+    function send(dataList: number[], length: number, reportID: number, resendUnchangedReport?: boolean): void;
 
     /** Sends an OutputReport to HID device
      *  @param reportID 1...255 for HID devices that uses ReportIDs - or 0 for devices, which don't use ReportIDs
      *  @param dataArray Data to send as byte array (Javascript type Uint8Array)
```

Comment:
Redundant and possibly wrong...
```suggestion
     *  @param dataArray Data to send as byte array
```